website/source/docs/partnerships/index.html.md

```diff
@@ -0,0 +1,137 @@
+
+## Development Process
+
+The Consul integration development process is described in the steps below. By following these steps, Consul integrations can be developed alongside HashiCorp to ensure new integrations are reviewed, approved and released as quickly as possible.
+
+1.  Engage: Initial contact between vendor and HashiCorp
+2.  Enable: Documentation, code samples and best practices for developing the integration
+3.  Develop and Test: Integration development and testing by vendor
+4.  Review/Certification: HashiCorp code review and certification of integration
+5.  Release: Consul integration released
+6.  Support: Ongoing maintenance and support of the integration by the vendor.
+
+### 1. Engage
+
+Please begin by completing [Consul Integration Program webform](https://docs.google.com/forms/d/e/1FAIpQLSf-RyVR9F0lmosao8Nnur0TTDjnl99gttnK3QP1OkfRefVKSw/viewform) to tell us about your company and the Consul integration you are developing.
+
+### 2. Enable
+
+Here are links to resources, documentation, examples and best practices to guide you through the Vault integration development and testing process:
+
+#### Data Plane:
+
+**Load Balancer**
+
+* [Load Balancing with NGINX and Consul Template](https://learn.hashicorp.com/consul/integrations/nginx-consul-template)
+* [Load Balancing with HAProxy Service Discovery](https://learn.hashicorp.com/consul/integrations/haproxy-consul)
+
+**Proxy**
+
+* [How to Integrate a Sidecar Proxy Documentation](https://www.consul.io/docs/connect/proxies/integrate.html)
+* [Example of Envoy Integration](https://www.consul.io/docs/connect/proxies/envoy.html)
+* [Consul’s source code](https://github.com/hashicorp/consul/blob/master/INTERNALS.md)
+
+**Firewall**
+
+* [Consul Connect Intentions](https://www.consul.io/docs/connect/intentions.html)
+* [Consul Connect Intentions Command Line](https://www.consul.io/docs/commands/intention.html)
+* [Consul Connect Intentions API](https://www.consul.io/api/connect/intentions.html)
+
+**API Gateway**
+
+* [Ambassador Integration documentation](https://www.consul.io/docs/platform/k8s/ambassador.html)
+
+#### Platform:
+
+* [Consul-AWS for AWS Cloud Map](https://learn.hashicorp.com/consul/integrations/consul-aws)
+* [Consul Agent Cloud Auto-joining](https://www.consul.io/docs/agent/cloud-auto-join.html)
+
+#### Infrastructure:
+
+**Certificate Authority (CA)**
+
+* [Consul Certificate Management Documentation](https://www.consul.io/docs/connect/ca.html)
+* [Securing RPC Communication with TLS Encryption](https://learn.hashicorp.com/consul/security-networking/certificates)
+* [Consul Connect CA API](https://www.consul.io/api/connect/ca.html)
+
+**Identity and Access Management (IAM)**
+
+* [ACL Documentation and Guides](https://www.vaultproject.io/docs/configuration/storage/index.html)
```

Comment:
Not sure you meant to include this link to the Vault docs?

Reply:
Thanks! fixed!